server/persistence/mongodb/qa/main.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
I was using this file to test without running unit tests. We can remove now that unit testing is in place